manual/guides/getting_serpent_license.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: manual
+title: Acquiring a Serpent License
+subtitle: "Step-by-step instructions for getting a Serpent license"
+permalink: /manual/guides/getting_serpent_license
+---
+
+# Register at RSICC
+
+- Go to [rsicc.ornl.gov](https://rsicc.ornl.gov/)
+- Click on "Registration"
+- Fill out the registration form. Once this is done,
+you should receive an RSICC Pass Number. You will
+need this for the next step.
+
+# Request the Software at RSICC
+
+- Go to [rsicc.ornl.gov](https://rsicc.ornl.gov/)
+- Go to "Requests"
+- Log in and fill out the request form.
+
+# The Request Form
+
+The request form is where you state your reason
+for needing Serpent. You are going to need the
+sponsoring professor(s), the professor(s) University
+email, your University email, and the course
+name(s)/number(s) with their description
+(or your project description if you are doing
+your own research).
+
+For the project description, there is a general template
+below that has been used by people that obtained their
+license:
+
+```markdown
+I am a graduate student in the Advanced Reactors and Fuel Cycles group at the University
```

Comment:
```suggestion
I am a graduate student in the Advanced Reactors and Fuel Cycles group at the University of Illinois in Urbana-Champaign, which is led by Prof. Kathryn Huff.
```

Reply:
I would put it in the same line, otherwise, it looks different depending on the window size.